packages/toolkit/src/lib/react-query-service/misc/useBlogPosts.ts

```diff
@@ -69,6 +73,9 @@ const fetchBlogPosts = async () => {
           metadata[key] = value;
         }
       });
+
+      const draft = metadata.draft === "true" || false;
```

Comment:
I think it can be simplified to `const draft = metadata.draft === "true"`. It's gonna evaluate to either `true` or `false`.